docs_source_files/content/remote_webdriver/remote_webdriver_client.de.md

```diff
@@ -87,7 +99,22 @@ caps.version = 67
 driver = Selenium::WebDriver.for :remote, :url => "http://www.example.com", :desired_capabilities => caps
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+const { Builder } = require("selenium-webdriver");
+const chrome = require("selenium-webdriver/chrome");
+const width = 780;
+const height = 480;
+(async function helloSelenium() {
+    let driver = new Builder()
+        .forBrowser('chrome')
+        .setChromeOptions(
+            new chrome.Options().headless().windowSize({ width, height }))
```

Comment:
Sorry to be annoying, but we do not need these extra options in these basic examples. Could we please leave it like the other examples?